src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +41,72 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void 지하철_역_목록을_조회() {
+        // given
+        createStationByStationName("강남역");
+        createStationByStationName("광화문역");
+
+        // when
+        List<String> stationNames =
+                selectStations().jsonPath().getList("name", String.class);
+
+        // then
+        assertThat(stationNames).containsAnyOf("강남역", "광화문역");
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 삭제한다.")
+    @Test
+    void 지하철_역을_삭제() {
+        // given
+        String stationName = "강남역";
+        Long id = createStationByStationName(stationName).jsonPath().getLong("id");
+
+        // when
+        deleteStation(id);
+
+        // then
+        assertThat(stationName).containsOnlyOnce(stationName);
```

Comment:
```suggestion
                var stationNames = selectStations().jsonPath().getList("name", String.class);
        assertThat(stationNames).doesNotContain(stationName);
```
생성 후 삭제 요청을 했으니, 다시 조회했을 때는 존재하지 않는 것을 검증해보면 어떨까요?

Reply:
남겨주신 질문 관련해서도 위와 같이 검증 조건을 변경한다면, 실패하는 것을 확인하실 수 있을거예요.

테스트 간 서로 영향을 주게 되어서, 남아있는 데이터로 인해 검증에 실패할 수 있어요. 
위 두 테스트와 아예 별개인 신논현역을 name으로 주고 삭제 테스트를 한다면, 아마 성공할거예요!

Reply:
앗 삭제 시나리오 인수 테스트 부분을 다시 보니, 삭제했는지 다시 확인하는 부분이 이상하군요...
조회 API를 통해 역 목록을 조회해 오고 그 목록을 검증하는 로직으로 바꾸었습니다. 😁
작성하신 코드에 var 키워드를 사용하셨는데, 실무에서도 var가 많이 사용될까요? 가독성도 좋지만 유지보수에 힘들지 않을까 생각이 들어서요ㅠ

Reply:
> 남겨주신 질문 관련해서도 위와 같이 검증 조건을 변경한다면, 실패하는 것을 확인하실 수 있을거예요.
> 
> 테스트 간 서로 영향을 주게 되어서, 남아있는 데이터로 인해 검증에 실패할 수 있어요.
> 위 두 테스트와 아예 별개인 신논현역을 name으로 주고 삭제 테스트를 한다면, 아마 성공할거예요!

넵, 삭제 시나리오 인수 테스트 수정 후에 전체 테스트를 하면 검증에 실패하고 있고 역 이름을 다르게 해서 테스트 성공하도록 수정하였습니다. 😂
제 생각에는 인수테스트끼리는 서로 영향을 주면 안될 것 같은데 이 부분을 신경쓰지 않고 테스트를 작성할 수 있는... 테스트 간 서로 영향을 주지 않는 방법이 있을까요?

Reply:
아하! 저는 비즈니스 로직이 담긴 코드에서는 var를 잘 사용하지 않아요ㅎㅎ 이번엔 반복적인 형태의 테스트코드이기도 하고 샘플로 짠거라 활용해봤어용